microkernel.ks

```diff
@@ -166,6 +166,13 @@ EOMEI
 echo " * compressing cracklib dictionary"
 gzip -9 /usr/share/cracklib/pw_dict.pwd
 
+echo " * setting up journald and tty1"
+rm -f /etc/systemd/system/getty.target.wants/getty@tty1.service
+echo "SystemMaxUse=15M" >> /etc/systemd/journald.conf
+echo "ForwardToSyslog=no" >> /etc/systemd/journald.conf
+echo "ForwardToConsole=yes" >> /etc/systemd/journald.conf
+echo "TTYPath=/dev/tty1" >> /etc/systemd/journald.conf
```

Comment:
Given the above comment, I think it makes sense to switch this to tty2. The output may still be useful, on a tty screen, but keeping the login page empty seems like a better goal.